tests/warp/warp_test.go

```diff
@@ -380,4 +383,111 @@ var _ = ginkgo.Describe("[Warp]", ginkgo.Ordered, func() {
 		gomega.Expect(err).Should(gomega.BeNil())
 		gomega.Expect(receipt.Status).Should(gomega.Equal(types.ReceiptStatusSuccessful))
 	})
+
+	ginkgo.It("Send Message from A to B from hardhat", ginkgo.Label("Warp", "WarpContract", "SendWarpContract"), func() {
```

Comment:
probably could live in a separate file if possible?

Reply:
They share a lot of same variables from `var _ = ginkgo.Describe("[Warp]", ginkgo.Ordered, func() {`